lms/templates/course_modes/track_selection.html

```diff
@@ -92,7 +92,11 @@ <h3 class="py-md-3">
 
                     <form class="form-register-choose mb-4 mt-0" method="post" name="enrollment_mode_form" id="enrollment_mode_form">
                         <div class="row">
-                        <div class="track-selection-options">
+                        % if is_verified_only or is_single_mode:
```

Comment:
I think this could just be `if is_single_mode:`

Reply:
True! Since `is_verified_only` is also `is_single_mode`, will update